arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -410,7 +410,7 @@
       "description": "Properties of Analysis Services resource.",
       "type": "object",
       "properties": {
-        "provisioningState": {
+        "state": {
```

Comment:
> Yes but we introduced one more "state" property in our ARM response to be used by service other than ARM. ProvisioningState is still there to be used by ARM.

Why create new things? Why can't the same property be reused to provide the same information?

> To be clear, "provisioningState" is still there, but we do not want it get used by SDK.

Why? The swagger spec should accurately describe the REST API. The spec is used for multiple purpose. It is used for generating REST API documentation (docs.microsoft.com) apart from generating the sdks. So describing the swagger spec accurately is super important.


I am still not convinced why is this change really required and why was the provisioningState not reused. I am sure there might be some valid reason behind this. Can you please explain that in detail so folks in the SDK team can get more context on this?